Makefile

```diff
@@ -44,7 +44,7 @@ update: ## Update Make and Buildout
 bin/buildout: bin/pip
 	bin/pip install --upgrade pip
 	bin/pip install -r requirements-5.2.txt
-	bin/pip install black || true
+	bin/pip install --upgrade click==$(awk '/^click =/{print $NF}' versions.cfg) black==$(awk '/^black =/{print $NF}' versions.cfg) || true
```

Comment:
Looks like black wasn't installed:

```shell
$ make black
if [ -f "bin/black" ]; then bin/black src/ ; fi
```

Reply:
Take a look at:

https://stackoverflow.com/questions/30445218/why-does-awk-not-work-correctly-in-a-makefile

Reply:
> Looks like black wasn't installed:
> 
> ```shell
> $ make black
> if [ -f "bin/black" ]; then bin/black src/ ; fi
> ```

but it shows black version `23.3.0`, so it means black is installed

Reply:
It must have been installed before you update the code in the Makefile. You can run a `make clean` to remove the previous installation. Then if you run `make` again you will see that `black` is not installed. The error occurs:]

```
bin/pip install --upgrade click== black== || true
ERROR: Could not find a version that satisfies the requirement click== (from versions: 0.1, 0.2, 0.3, 0.4, 0.5, 0.5.1, 0.6, 0.7, 1.0, 1.1, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 3.0, 3.1, 3.2, 3.3, 4.0, 4.1, 5.0, 5.1, 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7.dev0, 6.7, 7.0, 7.1, 7.1.1, 7.1.2, 8.0.0a1, 8.0.0rc1, 8.0.0, 8.0.1, 8.0.2, 8.0.3, 8.0.4, 8.1.0, 8.1.1, 8.1.2, 8.1.3, 8.1.4)
ERROR: No matching distribution found for click==
```